announcements/2021/week05.md

```diff
@@ -0,0 +1,12 @@
+# :zap: This week in provider-independent announcements (January 29, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[External DNS v2.0.0](waiting for link) allows customers to install and use External DNS for their own purpose, separate from the preinstalled app in AWS and Azure clusters. :warning: [Requires NGINX IC v1.14.0+](https://github.com/giantswarm/external-dns-app#limitations).
```

Comment:
@glitchcrab @giantswarm/app-squad-external-dns pls check!

Reply:
I bumped external-dns version because I had to release a fix for China, but then that is only a fix for us running it as a default App so I've reverted back to 2.0.0 - the release notes there are far more important for customers to see. Looks good to me.

Reply:
thanks!

Reply:
providing link as well.